package.json

```diff
@@ -26,13 +27,15 @@
   "devDependencies": {
     "autoprefixer": "^6.3.7",
     "broccoli-asset-rev": "^2.4.2",
+    "coveralls": "^2.11.16",
```

Comment:
Hmm. Strictly speaking the yarn lockfile will pin this rather explicitly in practice, if someone chooses to install that way. 

Have we made an official decision to pin *all* dependencies going forward, then? I knew we were versioning EOSF, but would be interested to know the details of any larger policy change.